gatsby-config.js

```diff
@@ -8,6 +8,16 @@ module.exports = {
   plugins: [
     'gatsby-plugin-react-helmet',
     'gatsby-plugin-emotion',
+    {
+      resolve: 'gatsby-plugin-matomo',
+      options: {
+        siteId: '5',
+        matomoUrl: '//analytics.octopusth.ink/',
```

Comment:
Our Analytics site is HTTPS-only... it might redirect but we should be doing this anyhow 😉 

```suggestion
        matomoUrl: 'https://analytics.octopusth.ink/',
```